src/Common.Projects.props

```diff
@@ -1,7 +1,7 @@
 <Project>
     <PropertyGroup>
         <Company>Trakx</Company>
-        <Copyright>Copyright © 2020 Trakx</Copyright>
+        <Copyright>Copyright © 2022 Trakx</Copyright>
```

Comment:
[suggestion] Change to 2023.

Reply:
https://github.com/trakx/kaiko-api-client/pull/53